README.md

```diff
@@ -35,28 +35,37 @@ To change these default directories simply run `armory configure` after installa
 
 # Usage
 
-There are four ways to interact with the armory container system.
+There are five ways to interact with the armory container system.
```

Comment:
Description looks great. I think it's more clear if we make this `3 ways`  to interact though and then have the different arguments described as sub-bullets.